### ISSUE TITLE
z80asm: inquiry about supporting the z280 cpu

### ISSUE THREAD

**Issue by aralbrec** [z88dk/z88dk#914]
From the board with some links to an example sbc:
https://www.z88dk.org/forum/viewtopic.php?id=10506

We have a datasheet in z88dk/techdocs:
https://github.com/z88dk/techdocs/raw/master/zilog/z280_cpu_tm.pdf

The z280 is a significant extension to the z80 so it may be some work to add to z80asm.

**Comment by pauloscustodio** [comment]
Started working on this and found a road-block: the z280 has two different opcodes for (ix+d), depending on the size of d (8 or 16 bits); this forces delaying the opcode selection to after the expressions are computed.

**Comment by pauloscustodio** [comment]
No, I started looking at it, but noticed a roadblock and there were no more requests,
